cadence/contracts/FUSD.cdc

```diff
@@ -1,33 +1,57 @@
-import FungibleToken from "./FungibleToken.cdc"
+import FungibleToken from "FungibleToken"
+import MetadataViews from "MetadataViews"
+import FungibleTokenMetadataViews from "FungibleTokenMetadataViews"
+import ViewResolver from "ViewResolver"
 
-pub contract FUSD: FungibleToken {
+access(all) contract FUSD {
```

Comment:
In conjunction with [comment below](https://github.com/onflow/faucet/pull/71/files#r1491315251), once we have access to an update EVM CLI pre-release with updated FT standards, we'll need to update this contract to implement `ViewResolver`